apps/engineeringblog/README.md

```diff
@@ -0,0 +1,36 @@
+This is a [Next.js](https://nextjs.org/) project bootstrapped with [`create-next-app`](https://github.com/vercel/next.js/tree/canary/packages/create-next-app).
```

Comment:
1. Update this README
2. Add this app to the list of apps in the main README at repo level
3. I think just `blog` is good enough... but, if we want to be technically correct, the team is called TechLab